packages/common/src/async.ts

```diff
@@ -1,3 +1,21 @@
+import { wait } from './util'
+
+export const readFromGenerator = async <T>(
+  gen: AsyncGenerator<T>,
+  maxLength = Number.MAX_SAFE_INTEGER,
+  timeout = 1000,
```

Comment:
bumping this up to 1000 let's all the tests pass when run in the test suite

I think PG just gets a bit bogged down & doesn't deliver notifcations in time